.github/workflows/main.yml

```diff
@@ -48,6 +48,7 @@ jobs:
 
```

Comment:
main.yml has been replaced by separate files: windows.yml, mingw.yml and android.yml. Please add these changes to those files.

Reply:
I updated the Windows build.